conf/test.config

```diff
@@ -40,3 +41,10 @@ params {
     namesdmp         = params.pipelines_testdata_base_path + 'createtaxdb/data/taxonomy/names.dmp'
     malt_mapdb       = 's3://ngi-igenomes/test-data/createtaxdb/taxonomy/megan-nucl-Feb2022.db.zip'
 }
+
+process {
+    withName:'KRAKENUNIQ_BUILD'{
+        memory = { check_max( 12.GB * task.attempt, 'memory'  ) }
+        ext.args = "--work-on-disk --max-db-size 14 --kmer-len 15 --minimizer-len 13 --jellyfish-bin \$(which jellyfish)"
```

Comment:
Nice with `--work-on-disk`. Did not know about that!

Reply:
Thanks to @Midnighter !